corehq/apps/geospatial/static/geospatial/js/geo_config.js

```diff
@@ -51,8 +52,26 @@ hqDefine("geospatial/js/geo_config", [
         });
 
         self.selectedAlgorithm = ko.observable();
+        self.minCasesPerUser = ko.observable(data.min_cases_per_user);
+        self.maxCasesPerUser = ko.observable(data.max_cases_per_user);
+
+        if (!data.max_cases_per_user) {
+            $("#div_id_max_cases_per_user").hide();
+        }
+
         self.plaintext_api_token = ko.observable(data.plaintext_api_token);
 
+        self.onSpecifyMaxCasesPerUser = function () {
+            var $maxCasesPerUserInput = $("#div_id_max_cases_per_user");
+            var $maxCasesPerUserCheckbox = $("#max_cases_per_user_checkbox_id");
+
+            if ($maxCasesPerUserCheckbox.is(":checked")) {
+                $maxCasesPerUserInput.show();
```

Comment:
nit: is it possible to do this via KO instead of jQuery? I believe such logic is more reliably put together with KO and you have a view model here already.